ambry-store/src/main/java/com/github/ambry/store/DiskManager.java

```diff
@@ -203,6 +202,7 @@ void start() throws InterruptedException {
       reportUnrecognizedDirs();
       running = true;
       if (diskHealthCheck.isEnabled()) {
+        logger.trace("Starting Disk Healthchecker");
```

Comment:
if we are only doing this for once, we can use info log
I think we can do a trace log on each test?

Reply:
Changed to include info log and trace log in each test